src/styles/blocks/body.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__image-img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-family: "Roboto", sans-serif;
```

Comment:
Remove 'font-family' declaration to use the inherited font from the rest of the application.